doc/user/install.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Install"
+description: "Get started with Materialize"
+menu: "main"
+weight: 2
+---
+
+You can access Materialize through the `materialized` binary, which you can install on macOS and Linux.
+
+## macOS installation
+
```

Comment:
For v0.1.0 we should mention that the binary release tarball only supports macOS Catalina, but building from source (which Homebrew will do automatically) will work.